vtex-sites/nextjs.store
# feat: Product releaseDate

## What's the purpose of this pull request?
This PR adds the product's release date to PDP's structured data. Also, this PR fixes two missing fields on Product's structured data. Below you can see a before/after comparison:
<div>
<img src="https://user-images.githubusercontent.com/1753396/184678542-9de9d31f-19f5-40e1-8d0b-0a7ed196fb9c.png" width="400px" /> 
<img src="https://user-images.githubusercontent.com/1753396/184678494-26e1698d-5369-43d2-a7ca-29483ae10ee6.png" width="400px" />
</div>

## How to test it?
Open [Google's rich results tool](https://search.google.com/test/rich-results) and run agains a product of this preview, e.g: https://preview-207--nextjs.preview.vtex.app/incredible-metal-chair-57860423/p. You should see the product's release date on the rich result
